db/migrate/20220722225402_create_account_debits.rb

```diff
@@ -0,0 +1,14 @@
+class CreateAccountDebits < ActiveRecord::Migration[6.0]
+  def change
+    create_table :account_debits do |t|
+      t.string :bank, null: false
+      t.string :branch
+      t.string :number, null: false
+      t.decimal :credit_limit, precision: 10, scale: 2, null: false, default: 0
+      t.string :owner_name
```

Comment:
_Minor suggestion:_ Maybe it's better to call this field as just `name` to avoid misunderstanding with the "group owner". What do you think?

```suggestion
      t.string :name
```